lib/publishing/npm/publish-npm.sh

```diff
@@ -22,6 +22,11 @@ fi
 # NPM #
 #######
 
+DISTTAG=${DISTTAG:-""}
```

Comment:
The `""` is unnecessary here.

Reply:
I know, but I wanted to be explicit about.